[K9VULN-4829] Update note to include inferable default branches (#29452)
DataDog/documentation: content/en/security/code_security/software_composition_analysis/setup_static/_index.md

## Generic CI Providers
If you don't use GitHub Actions, you can run the [datadog-ci][14] CLI directly in your CI pipeline platform and upload your SBOM to Datadog.

> **If you are running Code Security on a non-GitHub repository**, ensure that the first scan is ran on your default branch. If your default branch is not one of `master`, `main`, `default`, `stable`, `source`, `prod`, or `develop`, then you will need to attempt an SBOM upload for your repository and then manually override the default branch in-app under [Repository Settings][17]. Afterwards, uploads from your non-default branches will succeed.

Reviewer: ```suggestion
**If you are running Code Security on a non-GitHub repository**, ensure that the first scan is ran on your default branch. If your default branch is not one of `master`, `main`, `default`, `stable`, `source`, `prod`, or `develop`, you must attempt an SBOM upload for your repository and then manually override the default branch in-app under [Repository Settings][17]. Afterwards, uploads from your non-default branches will succeed.
```
